pkg/proxy/iptables/proxier.go

```diff
@@ -1100,17 +1096,18 @@ func (proxier *Proxier) syncProxyRules() {
 			// machine, hold the local port open so no other process can open it
 			// (because the socket might open but it would never work).
 			if (svcInfo.Protocol() != v1.ProtocolSCTP) && localAddrSet.Has(net.ParseIP(externalIP)) {
-				lp := utilproxy.LocalPort{
+				lp := utilnet.LocalPort{
 					Description: "externalIP for " + svcNameString,
 					IP:          externalIP,
+					IPFamily:    localPortIPFamily,
 					Port:        svcInfo.Port(),
-					Protocol:    protocol,
+					Protocol:    utilnet.Protocol(svcInfo.Protocol()),
```

Comment:
this is unfortunate, so we are using lowercase in all the `proxy` code but in the library we use uppercase

Reply:
That is the big differece between proxy/util/port.go and k8s.io/util/port.go. The other is LocalPort of k8s.io/util/port.go has new field ipfamily.